docs/src/main/asciidoc/user-guide/topics/import_files_to_panel.adoc

```diff
@@ -0,0 +1,284 @@
+                        </xsd:sequence>
+                      </xsd:complexType>
+                    </xsd:element>
+                  </xsd:sequence>
+                </xsd:complexType>
+              </xsd:element>
+              <xsd:element name="status" type="xsd:string" />
+            </xsd:sequence>
+          </xsd:complexType>
+        </xsd:element>
+      </xsd:schema>
+    </d:AdditionalSchemas>
+  </d:SchemaSet>
+```
+As was done in the previous JSON file import, select the import icon this time in the *Target* panel.
+
+image:ImportXML1.png[Import an XML File to Target Panel]
+
+An *Open File* dialog appears.  Select the location for XMLSchema.xml.
+The fields now appear in the *Target* panel as you've defined them in the imported XML schema.  The fields
+are expanded to show more detail.
+
+image:ImportXML2.png[Import of an XML File to Target Panel Complete]
+
+Instance files are imported in precisely the same manner.  These files define a separate namespace
+which also defines a few special attributes.  For example:
+
+* JSON Schema Instance File: *JSONSchemaInst.json*
+
+```
+   {
+        "order": {
+            "address": {
+                "street": "123 any st",
+                "city": "Austin",
+                "state": "TX",
+                "zip": "78626"
+            },
+            "contact": {
+                "firstName": "james",
+                "lastName": "smith",
+                "phone": "512-123-1234"
+            },
+            "orderId": "123"
+        },
+        "primitives": {
+            "stringPrimitive": "some value",
+            "booleanPrimitive": true,
+            "numberPrimitive": 24
+        },
+        "addressList": [
+            { "street": "123 any st", "city": "Austin", "state": "TX", "zip": "78626" },
+            { "street": "123 any st", "city": "Austin", "state": "TX", "zip": "78626" },
+            { "street": "123 any st", "city": "Austin", "state": "TX", "zip": "78626" },
+            { "street": "123 any st", "city": "Austin", "state": "TX", "zip": "78626" }
+        ]
+    }
+```
+* XML Schema Instance File: *XMLSchemaInst.xml*
+
+```
+ <ns:XmlOE xmlns:ns="http://atlasmap.io/xml/test/v2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://atlasmap.io/xml/test/v2 atlas-xml-test-model-v2.xsd ">
+    <ns:orderId>ns:orderId</ns:orderId>
+    <ns:Address>
+      <ns:addressLine1>ns:addressLine1</ns:addressLine1>
+      <ns:addressLine2>ns:addressLine2</ns:addressLine2>
+      <ns:city>ns:city</ns:city>
+      <ns:state>ns:state</ns:state>
+      <ns:zipCode>ns:zipCode</ns:zipCode>
+    </ns:Address>
+    <ns:Contact>
+      <ns:firstName>ns:firstName</ns:firstName>
+      <ns:lastName>ns:lastName</ns:lastName>
+      <ns:phoneNumber>ns:phoneNumber</ns:phoneNumber>
+      <ns:zipCode>ns:zipCode</ns:zipCode>
+    </ns:Contact>
+  </ns:XmlOE>
+```
+Using the same procedure as before, these files may be imported into either the *Source* or *Target*
```

Comment:
Instead:  ".....before, import instance files into either the *Source* or *Target* panel.